src/routes/travelRoute.js

```diff
@@ -5,30 +5,26 @@ import {
   pendingManagerApproval,
   getUserTravelStatus,
   approveTravelRequest,
+  mostTravelledDest,
+  userCanEditOpenRequest
 } from '../controllers/travelControllers';
+
 import {
   validateTravelRequest,
   validateResult,
 } from '../validation/travelValidation';
+
 import { verifyDeptManagerAndRequestStatus } from '../middlewares/travelsMiddleware';
 import { verifyRole } from '../middlewares/userMiddlewares';
 import { getToken, verifyToken } from '../middlewares/tokenMiddleware';
 
 const route = Router();
 
-const { 
-  createOneWayTrip,
-  pendingManagerApproval, 
-  getUserTravelStatus, 
-  mostTravelledDest
-  } = travelControllers;
-const { validateTravelRequest, validateResult } = travelValidator;
-
 // handles the api home route...
 route.post('/travel/one_way_trip', getToken, verifyToken, validateTravelRequest, validateResult, createOneWayTrip);
 
 // handles manager pending req approvals route
-route.get('/requests/pending/:manager', getToken, verifyToken, pendingManagerApproval);
+route.get('/travel/pending/:manager', getToken, verifyToken, pendingManagerApproval);
```

Comment:
I will suggest you use the manager's ID as a param instead of his/her name.
I will also suggest you use `/travel/pending_request/:manager_id` as the endpoint.

Reply:
@chuksjoe  Thank you for the suggestion

Yeah will change that to pending_request. 

Also thought of the manager's Id param but it is already in the token supplied, and all users will have their names as a param at a point.